src/pagination/pagination.spec.ts

```diff
@@ -519,6 +522,22 @@ describe('ngb-pagination', () => {
       expectPages(fixture.nativeElement, ['« Previous', '1', '+2', '-» Next']);
     });
 
+    it('should render and respond to direction links changes', () => {
+      const html = '<ngb-pagination [collectionSize]="collectionSize" [page]="1" [firstText]="firstText"' +
+          ' [previousText]="previousText" [nextText]="nextText" [lastText]="lastText"' +
+          ' [boundaryLinks]="boundaryLinks"></ngb-pagination>';
```

Comment:
It does not seem to me like you are talking here about moving the template to a template file. I am not exactly sure what you mean by a template string different than the one already included. Could you please point me to an example of a template string that I could use to model my change after?